exercises/beer-song/src/main/scala/BeerSong.scala

```diff
@@ -0,0 +1,5 @@
+class BeerSong {
```

Comment:
In #137 we agreed to just have a `.keep` file in the `src/main/scala` directory (except for the first couple of exercises) so that there is some additional fun to find out the function signatures. :)